css-three-column-layout/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="text-align-center ">
+      <div class="row">
+        <div class="column-full">
+          <header class=" font-size-bigger white-text"> Designed for life science.Optimized for SaMD and Digital Health.
+          </header>
+        </div>
+      </div>
+
+      <div class="column">
+        <div class="column-one-thirds">
+          <div class="circle">
+            <img class="size-pic" src="images/merge.png" alt="merge">
+          </div>
+          <div>
+            <p class="white-text"> Streamlined  QMS</p>
+          </div>
+          <div>
+            <p class="font-size-smallest light-grey-text">
+              A QMS that includes modules for all
+      stages of the product life cycle from
+      Design Control to CAPA</p>
+          </div>
+        </div>
+
+        <div class="column">
+          <div class="column-one-thirds">
+            <div class="circle">
+              <img class="size-pic" src="images/cloud-computing.png" alt="cloud-computing">
+            </div>
+            <div>
+              <p class="white-text"> Integrates with existing tools</p>
+            </div>
+            <div>
+              <p class="font-size-smallest light-grey-text">
+                Enzyme QMS integrates with your
+       existing tolls,such as JIRA,GitHub
+       and Jenkins with Enzyme</p>
+            </div>
+          </div>
+
+          <div class="column">
+            <div class="column-one-thirds">
+              <div class="circle">
+                <img class="size-pic" src="images/discussion.png" alt="discussion">
+              </div>
+              <div>
+                <p class="white-text">In-depth Onboarding</p>
+              </div>
+              <div>
+                <p class="font-size-smallest light-grey-text">
+                  We train your entire team and provide
+      you with support when and as its
+      needed</p>
```

Comment:
Your indentation seems a bit out there, please be very careful to keep your formatting correct.